web/src/firebase-config.js

```diff
@@ -8,6 +8,13 @@
  */
 const config = {
   /* TODO: ADD YOUR FIREBASE CONFIGURATION OBJECT HERE */
+  apiKey: "AIzaSyDgY1dBsJHB-pyxxrI1gvKOXgqy9qkxep8",
```

Comment:
I think you'll want to `commit --amend` to remove this file from the PR - this config is specific to the Firebase Project and associated web app, which developers will be filling out themselves

Reply:
We should have some fake data here which works in the emulator, so we only require developers to complete this step if they actually want to deploy. This makes setup much easier. This works for me:

```
  apiKey: "fake-api-key",
  projectId: "demo-example",
  authDomain: "demo-example.firebaseapp.com",
  appId: "fakeid"
```

(The last line is needed for FCM)